src/Microsoft.HttpRepl/Microsoft.HttpRepl.csproj

```diff
@@ -20,7 +20,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\Microsoft.Repl\Microsoft.Repl.csproj" />
+    <ProjectReference Include="$(RepoSource)Microsoft.Repl\Microsoft.Repl.csproj" />
```

Comment:
I find it much more readable to have the `\` separator after variables.  Can we keep the slash off in `$(RepoSource)` so it can be put here instead?

Reply:
I was trying to make it consistent with `$(RepoRoot)` which I can't change (Arcade uses it) and which has the slash. I agree about the readability though. I changed it, but I still don't like the inconsistency.